task_stages.go

```diff
@@ -0,0 +1,113 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+// Compile-time proof of interface  implementation
+var _ TaskStages = (*taskStages)(nil)
+
+// TaskStages describes all the task stage related methods that the TFC/E API
+// supports. **Note: This API is still in BETA and is subject to change.**
```

Comment:
Nit: as previously mentioned, placing this on it's own line could make it a cleaner diff to remove in future.